java-library/src/main/java/uk/co/lucasweb/aws/v4/signer/hash/Sha256.java

```diff
@@ -1,37 +0,0 @@
-/*
-  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
-  the License. You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
-  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
-  specific language governing permissions and limitations under the License.
-
-  Copyright 2016 the original author or authors.
- */
-package uk.co.lucasweb.aws.v4.signer.hash;
-
-import okio.Buffer;
-import uk.co.lucasweb.aws.v4.signer.SigningException;
-
-import java.nio.charset.Charset;
-
-/**
- * @author Richard Lucas
- */
-public final class Sha256 {
```

Comment:
This is the java implementation of SHA-256 which we replaced with a pure kotlin one.